pkg/request/request.go

```diff
@@ -0,0 +1,43 @@
+package request
+
+import (
+	"fmt"
+	"net/url"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+)
+
+func DeleteRequest(url string, noDryRun bool, connection *sdk.Connection) error {
+	request := connection.Delete()
+	err := ApplyPathArg(request, url)
+	if err != nil {
+		return fmt.Errorf("can't parse url '%s': %v\n", url, err)
+	}
+	if !noDryRun {
+		fmt.Printf("DRYRUN: Would have called %v if not passed.\n", request.GetPath())
```

Comment:
In my opinion, The "if not passed" portion is redundant. I would consider removing it.